### PR TITLE
Add delete multiple free time

### PR DESCRIPTION
**Pull request by larainezo**

### CODE REVIEW

**Review by nyankoclaws — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java [hunk 1]
```diff
@@ -81,30 +81,19 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
...
-            Integer newEnd = Integer.parseInt(trimmedFreeTimeTag.substring(9, 13));
-
-            for (FreeTimeTag tag : freeTimeTags) {
-                String trimmedTag = tag.toString().substring(1, tag.toString().length() - 1);
-                Integer currentStart = Integer.parseInt(trimmedTag.substring(4, 8));
-                Integer currentEnd = Integer.parseInt(trimmedTag.substring(9, 13));
-
-                if (!(trimmedTag.substring(0, 3).equals(day)) || !(newStart.equals(currentStart))
-                        || !(newEnd.equals(currentEnd))) {
-                    updatedTags.add(tag);
```
Comment (PallonCX) [inline src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java:107]: May I know why this part is removed? What does it intend to do previously, and why it doesn't need to be done now?

**Review by PallonCX — APPROVED** [review]
LGTM

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Review by donwong2308 — APPROVED** [review]
LGTM

**Pull request merged**